src/shared/seccomp-util.c

```diff
@@ -1747,3 +1749,174 @@ int seccomp_lock_personality(unsigned long personality) {
 
         return 0;
 }
+
+int seccomp_protect_hostname(void) {
```

Comment:
This function was probably backported by accident. 

I am fine with including it but we also need to backport corresponding parser, man page blurp, etc ...

Reply:
Right, this was by accident. I'm going to remove it.